app/models/assignment.rb

```diff
@@ -576,6 +577,8 @@ def get_detailed_csv_report_rubric
         else
           submission = grouping.current_submission_used
           final_result.push(submission.get_latest_result.total_mark / out_of * 100)
+          final_result.push(submission.get_latest_result.total_mark.to_s \
+                                                        +  '/' + out_of.to_s)
```

Comment:
Unnecessary spacing detected.<br>Operator `+` should be surrounded with a single space.